test/box-luatest/gh-5602-environment-vars-cfg_test.lua

```diff
@@ -0,0 +1,198 @@
+local t = require('luatest')
+local fio = require('fio')
+
+local justrun = require('luatest.justrun')
+local treegen = require('luatest.treegen')
+
+local g = t.group()
+
+local tmpdir = fio.tempdir()
+local listen_sock1 = fio.pathjoin(tmpdir, 'listen1.sock')
+local listen_sock2 = fio.pathjoin(tmpdir, 'listen2.sock')
+
+-- Check that environment cfg values are set correctly.
+g.test_gh_5602_tarantool_env_variables_1 = function()
+    local dir = treegen.prepare_directory({}, {})
+    local script_name = 'test_gh_5602_tarantool_env_variables_1.lua'
+    local script_body = ([[
+local test = require('tap').test('gh-5602')
+local status, err = pcall(box.cfg, {background = false, vinyl_timeout = 70.1})
```

Comment:
Nit: These options seems unneeded.

Reply:
Removed:

```diff
diff --git a/test/box-luatest/gh-5602-environment-vars-cfg_test.lua b/test/box-luatest/gh-5602-environment-vars-cfg_test.lua
index 41a6781185..8e4d938bc0 100644
--- a/test/box-luatest/gh-5602-environment-vars-cfg_test.lua
+++ b/test/box-luatest/gh-5602-environment-vars-cfg_test.lua
@@ -55,7 +55,7 @@ g.test_gh_5602_tarantool_env_variables_2 = function()
     local script_name = 'test_gh_5602_tarantool_env_variables_2.lua'
     local script_body = ([[
 local test = require('tap').test('gh-5602')
-local status, err = pcall(box.cfg, {background = false, vinyl_timeout = 70.1})
+local status, err = pcall(box.cfg)
 
 test:plan(6)
 test:ok(status, 'box.cfg is successful')
```

Reply:
There are other similar places below, but that's minor and I'll resolve the comment.